plugins/status/plugin.go

```diff
@@ -238,7 +238,10 @@ func (p *Plugin) Start(ctx context.Context) error {
 	p.manager.RegisterPluginStatusListener(Name, p.UpdatePluginStatus)
 
 	if p.config.Prometheus && p.manager.PrometheusRegister() != nil {
-		p.register(p.manager.PrometheusRegister(), pluginStatus, loaded, failLoad,
+		version := p.manager.Labels()["version"]
+		opaInfo := createOpaInfoGauge(version)
```

Comment:
We can avoid this by importing `github.com/open-policy-agent/opa/version` and then using the `Version` field.